contracts/light-clients/ics02-wrapper/interfaces/IICS02Precompile.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Note to self - it'd be good to have our interfaces published somewhere so that we can just use them rather than needing to copy into I files.